tests/conftest.py

```diff
@@ -29,13 +29,15 @@
 
 @pytest.fixture
 def test_data_file():
+    """when called, yeilds temporary file"""
```

Comment:
yeilds -> yields

Reply:
FYI If you have a single change like this you can [use the suggestion feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) — I just click **OK** and it gets committed. That way I can avoid opening an editor and just do it on my phone.
```suggestion
    """when called, yields temporary file"""
```